.drone.yml

```diff
@@ -5819,6 +5878,7 @@ trigger:
 
 depends_on:
   - corr/real-time-environment-cyclictest-latencies
+  - build-old-tezedge-binaries
```

Comment:
@akoptelov 
maybe I would add here also `build-tezedge-binaries` just to be explicit